zulipterminal/cli/run.py

```diff
@@ -108,12 +108,14 @@ def parse_zuliprc(zuliprc_str: str) -> Dict[str, Any]:
     while not path.exists(zuliprc_path):
```

Comment:
You probably want to update this commit title too? (pass on EOFError...)

Reply:
Yep, changed it